data/colors_v2/vars/component_dark.ts

```diff
@@ -3,7 +3,7 @@ import {alpha, darken, get} from '../../../src/utils'
 export default {
   avatar: {
     bg: alpha(get('scale.white'), 0.1),
-    border: alpha(get('scale.white'), 0.1),
+    border: get(’border.subtle’),
```

Comment:
Let's replace these directional single quotes with straight ones:

```suggestion
    border: get('border.subtle'),
```